mobile/src/navigations/main/main.tsx

```diff
@@ -27,12 +26,15 @@ const Main: React.FC = () => {
       <BottomTab.Screen
         name={MainScreenName.HOME}
         component={Home}
-        options={{ tabBarIcon: HomeIcon }}
+        options={{
+          tabBarIcon: HomeIcon,
+          header: ({ route }) => <Header title={route.name} />,
```

Comment:
```suggestion
          header: Header,
```
In Header component let's make `title` property optional and use `const { name } = useRoute();` by default